spec/system/admin/products_v3/products_spec.rb

```diff
@@ -130,16 +130,32 @@
 
       # create a product with a different supplier
       let!(:producer) { create(:supplier_enterprise, name: "Producer 1") }
-      let!(:product_by_supplier) { create(:simple_product, supplier: producer) }
+      let!(:product_by_supplier) { create(:simple_product, name: "Apples", supplier: producer) }
 
-      it "can search for a product" do
+      it "can search for and update a product" do
         visit admin_products_url
 
         search_by_producer "Producer 1"
 
         # expect(page).to have_content "1 product found for your search criteria."
         expect(page).to have_select "producer_id", selected: "Producer 1"
         expect_products_count_to_be 1
+
+        within row_containing_name("Apples") do
+          fill_in "Name", with: "Pommes"
+        end
+
+        expect {
+          click_button "Save changes"
+
+          expect(page).to have_content "Changes saved"
+          product_by_supplier.reload
+        }.to change { product_by_supplier.name }.to("Pommes")
+
+        # Search is still applied
+        # expect(page).to have_content "1 product found for your search criteria."
```

Comment:
Is this supposed to be commented out ?

Reply:
Sort of. Currently it says "1 product***s*** found for your search criteria". So technically this should be `pending`. 
It must be time to go ahead and fix that..